src/AddTodoForm.js

```diff
@@ -9,34 +10,36 @@ const AddTodoForm = ({ onAddTodo }) => {
     setTodoTitle(newTodoTitle);
   }
 
-
   const handleAddTodo = (event) => {
     event.preventDefault();
     if (!todoTitle) {
-      throw new Error("Can't be empty") 
+      throw new Error("Can't be empty")
```

Comment:
It's great that you're thinking about this! You will soon need to start gracefully handling all errors so it doesn't break your app. This still needs a bit of work but it's a good start.